rules/dispatch.go

```diff
@@ -124,15 +125,17 @@ func (r *DefaultRuleRenderer) hostDispatchChains(
 	}
 
 	var fromEndRules, toEndRules []Rule
-	if defaultFromChainName != "" {
+	if defaultChainName != "" {
 		// Arrange to goto the specified default chain for any packets that don't match an
-		// interface in the `endpoints` map.  (Currently we only use this for pre-DNAT
-		// policy, so it's only needed for 'from' chain programming; but we will extend
-		// later to other kinds of policy, and then it will be wanted equally in 'to' chain
-		// programming.)
+		// interface in the `endpoints` map.
 		fromEndRules = []Rule{
 			Rule{
-				Action: GotoAction{Target: defaultFromChainName},
+				Action: GotoAction{Target: defaultChainName},
+			},
+		}
+		toEndRules = []Rule{
+			Rule{
+				Action: GotoAction{Target: defaultChainName},
```

Comment:
@neiljerram we chatted about adding `ToHostDispatchChains` but I wasn't sure if that was necessary. I'm probably wrong but doesn't just adding `toEndRules` with the all-interfaces endpoint chain as the end rule work?